test/testutil/tests.c

```diff
@@ -454,8 +454,11 @@ static const char *print_time(const ASN1_TIME *t)
                                const char *s1, const char *s2,          \
                                const time_t t1, const time_t t2)        \
     {                                                                   \
-        ASN1_TIME *at1 = ASN1_TIME_set(NULL, t1);                       \
-        ASN1_TIME *at2 = ASN1_TIME_set(NULL, t2);                       \
+        ASN1_TIME *at1, *at2;                                           \
+        if (strcmp(#op, "eq") == 0 && t1 op t2)                         \
```

Comment:
op cannot be "eq", you meant opname probably however I see not point to limit this to the eq operation.

Reply:
Direct comparison is restricted to "eq" because I'm unsure if `time_t` is really comparable or not.

Reply:
It is an integer type. So it is comparable.

Reply:
Gotcha.